src/System.Threading.Tasks/tests/TaskScheduler/TaskSchedulerTests.cs

```diff
@@ -291,6 +291,7 @@ public static void RunSynchronizationContextTaskSchedulerTests_Negative()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Uses reflection to access an internal method of the TaskScheduler class.")]
```

Comment:
@joperezr The debugger will access this method using symbols, not reflection:
https://github.com/dotnet/corert/blob/master/Documentation/design-docs/diagnostics/diagnostics-tools-contract.md
It may be possible to modify these tests to access this method using symbols for UapAot, but I think it would be a nontrivial amount of work.